solidity/contracts/test/BondedECDSAKeepFactoryStub.sol

```diff
@@ -35,4 +35,35 @@ contract BondedECDSAKeepFactoryStub is BondedECDSAKeepFactory {
         /* solium-disable-next-line security/no-block-members*/
         keepOpenedTimestamp[keep] = block.timestamp;
     }
+
+    /// @notice Opens a new ECDSA keep.
+    /// @param _owner Address of the keep owner.
+    /// @param _members Keep members.
+    /// @param _tokenStaking Token staking address.
+    /// @param _keepFactory Keep factory address.
+    /// @param _creationTimestamp Keep creation timestamp.
+    ///
+    /// @return Created keep address.
+    function stubOpenKeep(
+        address _owner,
+        address[] memory _members,
+        address _tokenStaking,
+        address payable _keepFactory,
```

Comment:
I don't think we should have them as parameters. `BondedECDSAKeepFactory` has fields:
```
    TokenStaking tokenStaking;
    KeepBonding keepBonding;
```
and the values are passed in stub's constructor.

Reply:
Right, done.